docs/v1/details/tracking-llm-calls.mdx

```diff
@@ -3,8 +3,21 @@ title: "Tracking LLM Calls"
 description: "Tracking LLM Calls using the AgentOps SDK"
 ---
 
-When the AgentOps SDK detects the `openai` as an installable module, it will automatically
-start tracking OpenAI calls by instrumenting chat completions to track their usage.
+### How it works
+
+When the AgentOps SDK detects the `openai`, `litellm`, or `cohere` as installed modules, it will automatically
```

Comment:
I think we should maintain a list somewhere of models we support first-class and then do when you import openai or <link>any of the models we support first-class</link>. If we maintain this list separately we'll forget to update it and it'll cause confusion

Reply:
Let's just push this for now. We can do that in a later change